backend/attachRendererFiber.js

```diff
@@ -71,8 +71,9 @@ function getInternalReactConstants(version) {
     };
   }
   ReactSymbols = {
-    ASYNC_MODE_NUMBER: 0xeacf,
-    ASYNC_MODE_SYMBOL_STRING: 'Symbol(react.async_mode)',
+    CONCURRENT_MODE_NUMBER: 0xeacf,
+    CONCURRENT_MODE_SYMBOL_STRING: 'Symbol(react.concurrent_mode)',
+    DEPRECATED_ASYNC_MODE_SYMBOL_STRING: 'Symbol(react.async_mode)',
```

Comment:
This isn't sufficient. We need the number and string.

Reply:
The number is the same though?

Reply:
Haha, d'oh! Ok sorry.